config.lua

```diff
@@ -32,6 +32,8 @@ setting("string", "server", nil, true) -- Server address to connect to
 setting("number", "port", 6667) -- Server port to connect to
 setting("string", "NSPass") -- NickServ password
 setting("string", "sasl.user", irc.config.nick) -- SASL username
+setting("string", "username", irc.config.username) -- username
+setting("string", "realname", irc.config.realname) -- realname
```

Comment:
The default isn't set here properly, it should just be `"Minetest"`.

Also, the comments aren't helpful.  Something like `Username/ident` and `Real name/GECOS` would be slightly better.